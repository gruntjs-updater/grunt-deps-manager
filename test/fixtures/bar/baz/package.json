{
    "name": "bar-baz",
    "dependencies": {
        "pattern-dep": "0.1.0"
    },
    "devDependencies": {
        "some-dev-dep": "0.42.1000"
    }
}