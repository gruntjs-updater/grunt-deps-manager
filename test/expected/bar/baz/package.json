{
  "name": "bar-baz",
  "dependencies": {
    "pattern-dep": "0.0.8"
  },
  "devDependencies": {
    "some-dev-dep": "0.0.42"
  }
}